documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -14,33 +15,30 @@ MirrorMaker 2.0 uses:
 
 MirrorMaker 2.0 is based on the Kafka Connect framework, _connectors_ managing the transfer of data between clusters.
 
-MirrorMaker 2.0 uses the following connectors:
+MirrorMaker 2.0 consists of the following connectors:
 
-`MirrorSourceConnector`:: The source connector replicates topics from a source cluster to a target cluster.
+`MirrorSourceConnector`:: The source connector replicates topics from a source cluster to a target cluster. It also replicates ACLs and is necessary for the `MirrorCheckpointConnector` to run.
```

Comment:
Should we add a note that the ACL replication does not work with User Operator enabled?

Reply:
Is there a technical reason or is this potentially a feature that could be added?

Reply:
The User Operator has the KafkaUser CRs as the only source of truth. So if it is enabled, only two things will happen:
* It will rewrite the ACLs synced by MM2 with what is in the KafkaUser CRs
* The `KafkaUsers` will have the exact same ACLs configured, in which case they will be set by the User Operator and the ACL mirroring is not needed.

Reply:
I see, thanks for the explanation!